### PR TITLE
docs(weekly): add this week in databend 103

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

- Closes #issue

### CODE REVIEW

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Creating Network Policies
+
+**Network Policies** are options for managing network configurations of Databend services. You can use it to restrict access to your account based on user IP address.
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:45]: **Network Policies** are used to manage access to Databend services. They can also be utilized to restrict access to a SQL account based on users' IP addresses.

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+To create a set of network policies, you can use:
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:47]:
```suggestion
To create a set of network policies:
```

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+```SQL
+CREATE NETWORK POLICY <policy-name> ALLOWED_IP_LIST=(<allowed-ip>) BLOCKED_IP_LIST=(<blocked-ip>) COMMENT=<policy-comment>
+```
+
+To create or alter network policies for a specified user, you can use:
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:53]:
```suggestion
To restrict access to a SQL user account:
```

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [PR #12158 | feat(planner): Support profile table scan](https://github.com/datafuselabs/databend/pull/12158)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added `intersect_count`, `to_yyyymmddhh` function.
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:86]:
```suggestion
- Added `intersect_count` and `to_yyyymmddhh` functions.
```

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- Added `fuse_column` table function.
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:87]:
```suggestion
- Added `fuse_column` system function.
```

File: website/blog/2023-07-23-databend-weekly-103.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Renaming TSV file format to Text
+
+Databend supports a file format called TSV, which may mislead users into thinking that its only difference from CSV is the field delimiter.
+
+In reality, TSV corresponds to the text format of MySQL/Hive/PostgreSQL and uses escaping instead of quoting to handle delimiters in values (MySQL text supports quoting but not by default). ClickHouse also uses it for transferring data to/from MySQL.
+
+Therefore, we suggest renaming TSV as an alias for Text.
```
Comment (soyeric128) [inline website/blog/2023-07-23-databend-weekly-103.md:102]: 这句中文是想表达什么？

Comment (PsiACE) [reply website/blog/2023-07-23-databend-weekly-103.md]: Therefore, we suggest renaming TSV to TEXT.

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**